types/underscore/underscore-tests.ts

```diff
@@ -2500,9 +2513,11 @@ _.zip(['moe', 'larry', 'curly'], [30, 40, 50], [true, false, false]); // $Expect
     // tuple lists
     _.transpose(tupleList); // $ExpectType [string[], number[]]
     _(tupleList).transpose(); // $ExpectType [string[], number[]]
-    // This test fails for Typescript 4.6 and below only because it
-    // arbitrarily swaps the string[] | number[] union to number[] | string[]
-    // extractChainTypes(_.chain(tupleList).transpose()); // $ExpectType ChainType<[string[], number[]], string[] | number[]>
+    // Typescript 4.6 and below arbitrarily swaps the string[] | number[] union to number[] | string[],
+    // so we create a type alias to normalise the union.
+    type StringOrNumberList = string[] | number[];
```

Comment:
Why not lift the definition from line 2483 so you can reuse it here?

Reply:
Fair point, done.